Tools/Scripts/generate-gpup-webgl

```diff
@@ -1024,21 +1038,32 @@ def main():
 
     funcs = []
     unimplemented = []
+    cond = ""
     for fn in functions_input_fns:
         with open(fn) as f:
             lines = iter(f.readlines())
             for _ in read_lines_until(lines, "// Functions with a generated implementation. This list is used by generate-gpup-webgl script."):
                 pass
 
             for line in read_lines_until(lines, "// End of list used by generate-gpup-webgl script."):
+                m = re.match(r"\s*#\s*if\s+(.*)", line)
```

Comment:
this might match with end of input, skipping whitespace: `\s+$`

Reply:
I don't understand?